examples/material_gallery/lib/gallery/app.dart

```diff
@@ -5,8 +5,45 @@
 import 'package:flutter/material.dart';
 import 'package:flutter/scheduler.dart' show timeDilation;
 
+import '../demos.dart';
 import 'home.dart';
 
+final Map<String, WidgetBuilder> kRoutes = <String, WidgetBuilder>{
```

Comment:
Does the map constructor ensure that the keys are unique? If not we should assert as much